### ISSUE TITLE
Sonoff Basic ZBR3

### ISSUE THREAD

**Issue by recom273** [ebaauw/homebridge-hue#562]
#### Issue

As an experiment, I purchased a couple of the new Sonoff zigbee relays, they won’t connect to the raspbee - Is this because the shield doesn’t support every zigbee device? Or could it be a software issue?

The relay enters pairing mode, the bridge might be trying to pair as it stops flashing after a while.

#### Log Messages

```
```

**Comment by ebaauw** [comment]
Best ask this in the [deCONZ library](https://github.com/dresden-elektronik/deconz-rest-plugin).

Afaik, the RaspBee should support every ZigBee Home Automation (ZHA), ZigBee Light Link (ZLL), and ZigBee 3.0 device, that is: these devices should appear in the deCONZ GUI.  Most lights are supported out of the box by the REST API plugin, but other devices need to be supported explicitly, see the [WiKi](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Request-Device-Support).

**Comment by recom273** [comment]
Just reopening issue for general interest, maybe you want to add the relay to the list.

I am not too sure what happened, but I had an aqara mini switch that was not responding, I removed and added after a couple of resets, which is uncommon. 

When I finally got the switch added, the sonoff zigbee relay appeared as a light, I presumed it would be a switch.

If this helps anyone in the future or if you want to add it to the wiki ..
